_data/meltano/utilities/dbt-dry-run/dbt-dry-run.yml

```diff
@@ -0,0 +1,53 @@
+commands:
+  run:
+    args:
+      args:
+        --target ${DBT_DRY_RUN_TARGET}
+        --verbose
+      definition: |
+        Allows you to run the plugin. Requires that you specify a profile target.
+        Note - Requires you run `dbt-bigquery compile` before executing.
+      executable: dbt_dry_run_invoker
+
+definition: |
+  dbt-dry-run is a meltano plugin based on the awesome tool created by the team at AutoTraderUk.
+  It allows dbt-bigquery users to validate if their models will run (without encuring costs).
+  It leverages bigquery's [dry-run](https://cloud.google.com/bigquery/docs/dry-run-queries) feature.
+
+  **A note**
+  This plugin is at it's most powerful (in our opinion) in CI (we use github actions). It allows us to ensure that our changes will not break production.
+domain_url: https://github.com/autotraderuk/dbt-dry-run
+entity_url: https://cloud.google.com/bigquery/docs/dry-run-queries
+meltano_edk: true
+executable: dbt-dry-run
+keywords:
+- dbt
+- bigquery
+- testing
+- continuous-integrations
+label: dbt-dry-run
+logo_url: /assets/logos/utilities/dbt-dry-run.png
```

Comment:
@SBurwash - Following up from @tayloramurphy's comment ... if it's easier than adding a new logo, you are also welcome to reuse this existing dbt logo if you'd like - or anything else in that same directory.

https://github.com/meltano/hub/blob/main/_data/meltano/utilities/dbt-bigquery/dbt-labs.yml#L61

This is consistent (today at least) with all the dbt flavors using the same logo:

![image](https://user-images.githubusercontent.com/18150651/218584565-a9b933c3-dae7-4511-b304-8d1af38cb21c.png)

Reply:
<img width="241" alt="Screenshot 2023-02-14 at 11 29 14 AM" src="https://user-images.githubusercontent.com/5394188/218813274-efefe66c-2e88-425d-94bc-610f6c6556dc.png">

@SBurwash I mocked this up - what do you think?

Reply:
Awesome, thanks @tayloramurphy

Reply:
@SBurwash np! If you could just add this to the utility folder and update the yaml to reference it please 😄

Reply:
😍 PRETTY! 😄